config/local-dev.yml

```diff
@@ -7,7 +7,13 @@ analysis:
     metadata_registry:
       interval: 1h
     aggregate_stats:
-      interval: 10m
+      tx_volume_interval: 10m
+      daily_active_accounts_window_step: 5m
+      daily_active_accounts_layers:
```

Comment:
Not sure if it even makes sense for these to be configurable, could always index all layers.

Reply:
Yeah, the code should handle this just fine (only one `info` log message per missing layer per iteration would be emitted).

Will probably remove this from config.